programs/marginfi/src/instructions/marginfi_group/panic_unpause.rs

```diff
@@ -0,0 +1,51 @@
+use anchor_lang::prelude::*;
+use marginfi_type_crate::{constants::FEE_STATE_SEED, types::FeeState};
+
+use crate::state::panic_state::PanicStateImpl;
+
+pub fn panic_unpause(ctx: Context<PanicUnpause>) -> Result<()> {
```

Comment:
why even bother having an unpause if it only pauses for 15 minutes?

Reply:
Up to 30 with extensions. I suppose we don't strictly need it but a 30 minute pause could be very annoying to the consumers so ideally we make it as short as possible whenever needed.